manifests/server/instance/systemd.pp

```diff
@@ -0,0 +1,20 @@
+# @param service_name Overrides the default PostgreSQL service name.
+define postgresql::server::instance::systemd (
+  String[1] $service_name = $name,
+) {
+  # Template uses:
+  # - $facts['os']['name']
+  # - $facts['os']['release']['major']
```

Comment:
I always said you can drop the Fedora specific bits there. It never made sense, but didn't want to block the PR from moving forward.

Reply:
I also removed the service name, because the block with the conditon using it, was removed